### DISCUSSION TITLE
AppVeyor The build job does not contain any artifacts

### DISCUSSION THREAD

**Discussion by calebvanee04** [SaitDev/MobArenaExtension#14] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/87914633/126880685-03c4ba9d-b4e2-48fb-824d-596e79e086f1.png)

**Comment by SaitDev** [comment, marked as answer]
The artifact is only available for 30 days. Im adding an alternate option - github action https://github.com/SaitDev/MobArenaExtension/actions which has maximum 90 days available but requires you to login.

I will move stable releases to Github release asap for longer live
